anomalib/post_processing/visualizer.py

```diff
@@ -132,9 +133,9 @@ def _visualize_full(self, image_result: ImageResult):
         elif self.task == "classification":
             visualization.add_image(image_result.image, title="Image")
             if image_result.pred_label:
-                image_classified = add_anomalous_label(image_result.heat_map, image_result.pred_score)
+                image_classified = add_anomalous_label(image_result.image, image_result.pred_score)
             else:
-                image_classified = add_normal_label(image_result.heat_map, 1 - image_result.pred_score)
+                image_classified = add_normal_label(image_result.image, 1 - image_result.pred_score)
```

Comment:
I'm curious why this was changed to show the original image, instead of the heatmap one?
Even if one only wants to classify images, it's interesting to see the heatmap during development.

curiously enough, it's also inconsistent now, since in `_visualize_simple` (L:161) , the heatmap is still used

Reply:
Not all AD methods generate heat_maps (for example, ganomaly does not). Therefore, we need to distinguish between `classification` and `segmentation` tasks also in the `Visualizer`. The behavior for `segmentation` tasks should therefore be left unchanged (we check for task in L:133).

Regarding the second point: I might have missed changing it there/it might actually be a bug, should be investigated further.